src/sgame/sg_bot.cpp

```diff
@@ -35,28 +35,22 @@ static AITreeList_t treeList;
 Bot management functions
 =======================
 */
-static struct
-{
-	int count;
-	struct
-	{
-		char *name;
-		bool inUse;
-	} name[MAX_CLIENTS];
-} botNames[NUM_TEAMS];
+struct nameInfo_t {
+	std::string name;
+	bool inUse;
+};
+static BoundedVector<nameInfo_t, MAX_CLIENTS> botNames[NUM_TEAMS];
```

Comment:
I don't think there's any reason for this to be fixed-size. It's no worse than the current code though, so I'm not demanding to rewrite.

Reply:
The initial reason was that the gentity_t was being memset. A reason that is not valid anymore indeed.